### DISCUSSION TITLE
untangle back to XML

### DISCUSSION THREAD

**Discussion by orellabac** [stchris/untangle#120] [Q&A] [answered]
Hi, is there any option to untangle back to a XML

I was planning to use XML to collect some information and then modify and save back to xml.

How can I do that? Any recomendations?

**Comment by stchris** [comment, marked as answer]
Hi @orellabac. Sorry for taking so long to respond. `untangle` is a one-way street from XML to Python objects. Other libraries might be what you are looking for, including https://pypi.org/project/xmltodict/
